util/MySqlMigrations/Migrations/20240913205322_2024-09-13_00_VerifiedOrgDomainSsoDetails.cs

```diff
@@ -0,0 +1,21 @@
+﻿using Microsoft.EntityFrameworkCore.Migrations;
+
+#nullable disable
+
+namespace Bit.MySqlMigrations.Migrations;
+
+/// <inheritdoc />
+public partial class _20240913_00_VerifiedOrgDomainSsoDetails : Migration
+{
+    /// <inheritdoc />
+    protected override void Up(MigrationBuilder migrationBuilder)
```

Comment:
You don't need to generate Entity Framework migrations for these kinds of changes. EF migrations are only applicable to schema changes like adding a new table of column. Updating a `READ` stored procedure in Dapper means you actually need to update the `where` clause in [this LINQ query](https://github.com/bitwarden/server/blob/f72932bf2496e70776a5fb81f0d4734b3807da9d/src/Infrastructure.EntityFramework/Repositories/OrganizationDomainRepository.cs#L82) that is an in-code mirror of the SQL stored procedure. If you update that you can delete these empty migrations.